src/content/docs/apm/agents/manage-apm-agents/agent-data/triage-run-diagnostics.mdx

```diff
@@ -52,18 +61,84 @@ After a customer puts a pair of shoes in their cart and decides it's time to che
 
 To figure out why customers can't click the `Pay Now` button, open APM and select your `Checkout-service`. 
 
```

Comment:
Just a copy note: we drop the `-` in any e-word, i.e. ecommerce, email, esports, ebook, etc

Reply:
(does not escape me that we use a hyphen in `e-word` despite not using the `-` for the words themselves